code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1583,9 +1583,13 @@
 
 	sleep(sleep_amount)
 
-	visible_message(SPAN_DANGER("[src] has successfully removed the acid!"), \
-			SPAN_NOTICE("You get rid of the acid."), null, 5)
-	extinguish_acid()
+	if( extinguish_acid() )
+		visible_message(SPAN_DANGER("[src] has successfully removed the acid!"), \
+				SPAN_NOTICE("You get rid of the acid."), null, 5)
```

Comment:
i'm declaring war on null args
you WILL max_distance = 5

Reply:
fixed